core/templates/dev/head/components/alerts/AlertMessageDirective.js

```diff
@@ -33,14 +33,20 @@ oppia.directive('alertMessage', [function() {
     ],
     link: function(scope) {
       var message = scope.getMessage();
+      var timeoutVal = 1500;
       if (message.type === 'info') {
+        if (message.timeout !== 1500 && message.timeout !== undefined) {
```

Comment:
In the if condition, firstly check whether its defined or not and then check the message timeout. Also, its better if you  write `timeoutVal` instead of 1500 in the if condition.

Reply:
Is checking that its undefined even required here?. I think it will always have some value, because of the checker in the  addInfoMessage function, which assigns 1500 in case timeout(parameter passed) is undefined.

Reply:
Sure :+1: